jobs/jobs.go

```diff
@@ -63,6 +66,16 @@ func Start() {
 		logger.Errorf("Failed to schedule job for cleaning up event queue table: %v", err)
 	}
 
+	if _, err := ScheduleFunc(CleanupNotificationSendHistorySchedule, func() {
+		if count, err := duty.DeleteNotificationSendHistory(api.NewContext(db.Gorm, nil), 30); err != nil {
```

Comment:
nit: The logging of the results should probably be in the function itself, and we should be passing through a DbContext rather than db.Gorm

Reply:
> The logging of the results should probably be in the function itself

I am not sure if the duty func should be doing the logging.

> we should be passing through a DbContext rather than db.Gorm

All of the jobs use `db.Gorm` directly and there's no DBContext at this stage yet. I guess we could pass gorm directly to the jobs starting function

```
func Start() {}
```

to
```
func Start(db *gorm.DB) {}
```